tests/common/config_reload.py

```diff
@@ -77,4 +79,8 @@ def config_reload(duthost, config_source='config_db', wait=120, start_bgp=True,
     modular_chassis = duthost.get_facts().get("modular_chassis")
     wait = max(wait, 240) if modular_chassis else wait
 
-    time.sleep(wait)
+    if safe_reload:
+        assert wait_until(wait, 20, 0, duthost.critical_services_fully_started), \
```

Comment:
Use pytest_assert instead?

Reply:
The wait time is apparently not enough, you need to wait longer. e.g. wait + 300.

Reply:
Done.